jobs/ci-run/unittest/unittests.yml

```diff
@@ -73,19 +65,12 @@
     - string:
         name: USE_TMPFS_FOR_MGO
         description: "Set to 1 to use a tmpfs volume for hosting mongo data"
-    - string:
-        name: BOOTSTRAP_SERIES
-        default: ""
-        description: "Series to used for bootstrapping"
-    - string:
-        name: GOVERSION
-        default: ""
-        description: "Go version to use for unit tests"
     builders:
       - wait-for-cloud-init
       - install-common-tools
       - apt-install:
           packages: gcc squashfuse
+      - detect-commit-go-version
```

Comment:
instead of using github api, move get-s3-source-payload up to here, then install go